core/configs/ledgers/indy/__mocks__/fixtures/sovrin-main-net-pool.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Do we want to be introducing test artifacts outside of the `__test__/` folder? Just concerned that it will get forgotten about.

Reply:
Putting `__mocks__` where it is was suggested by the Jest docs. It seemed reasonable to put the data close it the code that uses it. Its not used by the tests but rather a faux ledger returned by the mock. I don't want to use real ones because they'll be updated periodically and break tests. I can move it inside `__mocks__`. Thoughts?